enterprise/backend/test/metabase_enterprise/sandbox/query_processor/middleware/row_level_restrictions_test.clj

```diff
@@ -680,7 +682,7 @@
             (is (= [[10]]
                    (mt/rows result)))))
         (testing "Run the query with different User attributes, should not get the cached result"
-          (mt.tu/with-user-attributes :rasta {"cat" 40}
+          (mt/with-user-attributes :rasta {"cat" 40}
```

Comment:
namespace alias fix. We can get this from `mt`. No need for the more specialized `mt.tu`